src/hooks/usePlayerList.ts

```
@@ -0,0 +1,140 @@
+              ? entries.sort((a, b) => b.lastActive - a.lastActive)[0]
+              : { displayName: 'Unknown' };
+
+          const { displayName = 'Unknown' } = mostRecentEntry;
+
+          const userGameMessage = uniqueGameActions.find(
+            (message) => message.uid === onlineUid
+          )?.text;
+          const locationRegEx = /(?:#)[\d]*(?=:)/gs;
+          let currentLocation = 0;
+
+          if (userGameMessage && userGameMessage.match(locationRegEx)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)